python-shell/src/fishbowl/fishbowl.py

```diff
@@ -74,22 +75,54 @@ def _generate_library(self):
         binary_operators_python = self._generate_aggregation_functions()
         config_python = self._generate_config()
 
-        self._write_to_file(os.path.join(self.generated_directory_path, "functions.py"), functions_python)
-        self._write_to_file(os.path.join(self.generated_directory_path, "predicates.py"), predicates_python)
-        self._write_to_file(os.path.join(self.generated_directory_path, "binary_operators.py"), binary_operators_python)
-        self._write_to_file(os.path.join(self.generated_directory_path, "operations.py"), operations_python)
-        self._write_to_file(os.path.join(self.generated_directory_path, "config.py"), config_python)
-        self._write_to_file(os.path.join(self.generated_directory_path, "__init__.py"),
-                      "__all__ = [ \"operations\", \"predicates\", \"functions\", \"binary_operators\", \"config\" ]\n")
+        self._write_to_file(
+            os.path.join(
+                self.generated_directory_path,
+                "functions.py"),
+            functions_python)
+        self._write_to_file(
+            os.path.join(
+                self.generated_directory_path,
+                "predicates.py"),
+            predicates_python)
+        self._write_to_file(
+            os.path.join(
+                self.generated_directory_path,
+                "binary_operators.py"),
+            binary_operators_python)
+        self._write_to_file(
+            os.path.join(
+                self.generated_directory_path,
+                "operations.py"),
+            operations_python)
+        self._write_to_file(
+            os.path.join(
+                self.generated_directory_path,
+                "config.py"),
+            config_python)
+        self._write_to_file(
+            os.path.join(
+                self.generated_directory_path,
+                "__init__.py"),
+            "__all__ = [ \"operations\", \"predicates\", \"functions\", \"binary_operators\", \"config\" ]\n")
```

Comment:
This is a step backwards in terms of readability. It is caused by the default line wrapping of 79 characters. [This can be fixed and this change will no longer happen.](https://github.com/gchq/gaffer-tools/pull/1028/files#r991260378)